src/pgroonga.c

```diff
@@ -6119,6 +6127,14 @@ PGrnSearchBuildCondition(Relation index, ScanKey key, PGrnSearchData *data)
 		PGrnSearchBuildConditionIn(data, key, targetColumn, attribute);
 		return;
 	}
+	if (PGrnSearchIsMatchInCondition(key))
+	{
+		// PostgreSQL 18 optimaize to "column IN (keyword1, keyword2, ...)" from
+		// "column %% keyword1 OR column %% keyword2 OR ...".
+		// %% is match operator. So, we handle "column %% keyword1 OR column %%
+		// keyword2 OR ..." as "match in" operator.
```

Comment:
Is it really `IN`? `column &@ ANY (keyword1, keyword2, ...)`?

BTW, why did you use deprecated `%%` not `&@`?